travis/test_measures.sh

```diff
@@ -6,6 +6,8 @@ if ! ./travis/rebuild_bundles.sh; then
 fi
 
 cd fhir-patient-generator
+git checkout ci-param-tool-rebase
```

Comment:
```suggestion
git checkout master
```
Did you mean to change this to `master`? That other branch was merged and deleted.

Reply:
Oh wait no `checkout master` would be redundant, so I guess did you want to remove this?